keda/Chart.yaml

```diff
@@ -8,7 +8,7 @@ kubeVersion: ">=v1.24.0-0"
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.10.2
```

Comment:
Could you revert this change and remove the binary? as we serve multiple charts from here, we ship the releases during the release process